[JBTM-3805] Introducing DB allocator into our CI (#2186)
jbosstm/narayana: qa/README.txt

For jdbc tests, ensure the required drivers are present (cd qa && mvn install)
and create a suitable config/jdbc_profiles/<name_of_testnode_host>/ file by copying the existing

> config/jdbc_profiles/_template/ directory or rely on the one in config/jdbc_profiles/default and fill the database information

Author: Sure, that is a good idea. I have updated it, thanks

(e.g. username, password, host and port) 
By convention each test node has two accounts on each database server, with names of '<testnode_hostname>1'
and '<testnode_hostname>2'. This allows for testing of transactions with two resources. These accounts
need table creation privs plus the usual CRUD. The actual tables will be created automatically when the
tests run.

> Please note that if the parameter 'profile' is passed as a ant parameter (e.g. -Dprofile=postgres)

Reviewer: ```suggestion
Please note that if the parameter `profile` is passed as an `ant` parameter (e.g. `-Dprofile=postgres`)
```

Author: Thanks, I will update it.

> then QA junit-jdbc-testsuite will run against the database profile specified only.

Reviewer: ```suggestion
then QA will only run against the specified database profile.
```

Note: I don't think mentioning `junit-jdbc-testsuite` adds clarity to the text

Author: Thanks Manuel for your suggestion. I will apply your commit and I will align  junit-jdbc-crachrec and jta ant targets in order to respect that behaviour: "QA will only run against the specified database profile"
